components/CreateFlow.js

```diff
@@ -0,0 +1,140 @@
+import { nanoid } from "nanoid";
+import { useState } from "react";
+import styled from "styled-components";
+
+export default function CreateFlowForm({ flows, setFlows, setOpen }) {
```

Comment:
it would be better if this component received a function that can only add a flow instead of being able to manipulate the flow at will. In other words: an addFlow function would be better as a prop than a setFlows function.

Reply:
The setOpen could also be handled outside (inside the addFlow function), so this component does not have to be trusted to use the function after adding a flow.